src/Exceptions/AbstractMultiErrorException.php

```diff
@@ -9,84 +9,50 @@
 use MyParcelCom\JsonApi\Exceptions\Interfaces\MultiErrorInterface;
 use Throwable;
 
+/**
+ * @property JsonSchemaErrorInterface[] errors
```

Comment:
This is not public property though, therefore this doc annotation is misleading.

Perhaps you can add this in the constructor docblock instead.